atecc608a/main.c

```diff
@@ -488,12 +503,12 @@ bool interactive_loop()
                    slot_private, slot_public);
             return false;
         }
-        setup_key_attributes(&public_attributes, slot_public, 0);
```

Comment:
Was this tested with a different public key slot? Shouldn't new public key attributes be set up before the exporting?

Reply:
> Shouldn't new private key attributes be set up before the exporting?

This would have to be done with `mbedtls_psa_register_se_key()` before it is possible to ask PSA to export the key.

Have we been missing a `mbedtls_psa_register_se_key()` in our `interactive_loop()`?